scalafix-tests/unit/src/test/scala/scalafix/tests/interfaces/ScalafixArgumentsSuite.scala

```diff
@@ -66,22 +64,25 @@ class ScalafixArgumentsSuite extends AnyFunSuite with DiffAssertions {
   val main: Path = src.resolve("Main.scala")
   val relativePath: Path = cwd.relativize(main)
 
+  val specificScalacOption2: Seq[String] =
+    if (!ScalaVersions.isScala3)
+      Seq(removeUnused)
+    else Nil
+
```

Comment:
@mlachkar I thought we were handling scalacOptions in the [CompatSemanticdb](https://github.com/scalacenter/scalafix/pull/1650/files#diff-4cc852444bb821de2b756471dc5f8040c8a634fabf8c91e2ef4629229eb21cbc) classes in `scala2` and `scala3`. That's where I already moved `"-Yrangepos"`

Reply:
Actually this is not about semanticdb so I would keep it separated, we'll have to reconsider that when working on https://github.com/scalacenter/scalafix/issues/1680 anyway